client/src/components/Workflow/HistoryInvocations.vue

```diff
@@ -0,0 +1,34 @@
+<template>
+    <div>
+        <InvocationsList
+            v-if="currentUser && historyName"
+            :user-id="currentUser.id"
+            :history-id="historyId"
+            :history-name="historyName" />
+    </div>
+</template>
+<script>
+import InvocationsList from "components/Workflow/InvocationsList";
+import { mapState } from "pinia";
+import { useUserStore } from "@/stores/userStore";
+import { useHistoryStore } from "@/stores/historyStore";
+
+export default {
```

Comment:
We are trying to move towards Composition API + Typescript for new components. This one seems easy enough, would you mind converting it ;)